src/dynamicprompts/wildcards/wildcard_manager.py

```diff
@@ -122,8 +124,9 @@ def get_collection_names(self) -> set[str]:
     def get_all_values(self, wildcard: str) -> list[str]:
         """
         Get all wildcard values matching the given wildcard pattern.
+        Caching disabled if wildcard manager is configured to shuffle wildcards.
```

Comment:
I think it'd be better to just keep caching things, but shuffle a copy of `list_values` just before returning it from here?